lib/pry/input_completer.rb

```diff
@@ -239,4 +227,39 @@ def build_path(input)
     end
     return path, input
   end
+
+  private
+
+  def self.fetch_obj_space_collection
```

Comment:
Got it. On the other hand completer instance is connected to binding, and we would need to update it on every switch. IMO This would not be good design.